src/report/mod.rs

```diff
@@ -336,9 +347,12 @@ pub fn write_pdf_detail(
     diffs: &[(usize, String)],
     rule_name: &str,
 ) -> Result<FileCompareResult, Error> {
+    let compared_file_name = get_relative_path(actual.as_ref(), nominal.as_ref())
+        .to_string_lossy()
+        .to_string();
     let mut result = FileCompareResult {
-        nominal: get_file_name(nominal.as_ref())?.to_string(),
-        actual: get_file_name(actual.as_ref())?.to_string(),
+        nominal: compared_file_name.clone(),
```

Comment:
if we only clone them always, do we then really need to have it twice?

Reply:
I wasn't sure whether to keep the Struct as it is, or just modify it to have it only 1. I guess modifying it is more cleaner. I will change it